api/models.py

```diff
@@ -201,6 +201,10 @@ class Node(DatabaseModel):
         ge=0.0,
         le=24.0
     )
+    debug: Optional[Dict] = Field(
+        description='Additional debug fields. Can be ignored for all \
```

Comment:
Need to mention what kind of debug information will be stored here e.g any kind of URL or logs.